addon/components/base/bs-form.js

```diff
@@ -136,6 +137,31 @@ export default Component.extend({
    */
   horizontalLabelGridClass: 'col-md-4',
 
+  /**
+   * `isSubmitting` is `true` after `submit` event has been triggered and until Promise returned by `onSubmit` is
+   * fulfilled. If `validate` returns a Promise that one is also taken into consideration.
+   *
+   * `isSubmitting` property is yielded and could be used to customize form submit button. E.g. a loading spinner could
+   * be shown and the submit button been disabled:
+   *
+   * ```
+   * {{#bs-form onSubmit=(action 'save') as |form|}}
+   *   {{#bs-button type='submit' disabled=form.isSubmitting}}
+   *     Save {{fa-icon 'spinner'}}
+   *   {{/bs-button}}
+   * {{/bs-from}}
+   * ```
+   *
+   * The value is not accurate if multiple submit events are pending. In that case `isSubmitting` is `false` as soon
+   * as the first submission fulfills.
+   *
+   * @property isSubmitting
+   * @type {Boolean}
+   * @readonly
+   * @public
```

Comment:
I think I would rather declare this private, and move the docs for the yielded `isSubmitting` to the top to the component docs itself. The user is not really using this prop directly, it is just yielded, which is kind of like copying the value to some other context. We could for example rename the (private) prop here and still yield it under the same name...